idaes/power_generation/costing/power_plant_costing.py

```diff
@@ -0,0 +1,697 @@
+
+    IGCC_preloaded_accounts = {'Coal Handling': ['1.1', '1.2',
+                                                 '1.3', '1.4', '1.9'],
+                               'Coal Feed': ['2.1', '2.2',
+                                             '2.3', '2.4', '2.9'],
+                               'Feedwater System': ['3.1', '3.3'],
+                               'Gasifier': ['4.1'],
+                               'Syngas Cooler': ['4.2'],
+                               'ASU': ['4.3a'],
+                               'ASU Oxidant Compression': ['4.3b'],
+                               'Combustion Turbine': ['6.1', '6.3'],
+                               'Syngas Expander': ['6.2'],
+                               'HRSG': ['7.1', '7.2'],
+                               'Steam Turbine': ['8.1'],
+                               'Condenser': ['8.3'],
+                               'Cooling Tower': ['9.1'],
+                               'Circulating Water System': ['9.2', '9.3',
+                                                            '9.4', '9.6',
+                                                            '9.7'],
+                               'Slag Handling': ['10.1', '10.2',
+                                                 '10.3', '10.6',
+                                                 '10.7', '10.8',
+                                                 '10.9']}
+
+    NGCC_preloaded_accounts = {'Feedwater System': ['3.1', '3.3'],
+                               'Combustion Turbine': ['6.1', '6.3'],
+                               'HRSG': ['7.1', '7.2'],
+                               'Steam Turbine': ['8.1'],
+                               'Condenser': ['8.3'],
+                               'Cooling Tower': ['9.1'],
+                               'Circulating Water System': ['9.2', '9.3',
+                                                            '9.4', '9.6',
+                                                            '9.7']}
+
+    AUSC_preloaded_accounts = {'PC Boiler': ['4.9'],
+                               'Steam Turbine': ['8.1'],
+                               'Steam Piping': ['8.4']}
+
+    # preloaded account handling
+    if type(cost_accounts) == str:
+        if tech in [1, 2]:
+            cost_accounts = PC_preloaded_accounts[cost_accounts]
+        elif tech in [3, 4, 5]:
+            cost_accounts = IGCC_preloaded_accounts[cost_accounts]
+        elif tech == 6:
+            cost_accounts = NGCC_preloaded_accounts[cost_accounts]
+        elif tech == 7:
+            cost_accounts = AUSC_preloaded_accounts[cost_accounts]
+
+    # check that all accounts use the same process parameter
+    process_params = set()
+    for account in cost_accounts:
+        param = BB_costing_exponents[str(tech)][account]['Process Parameter']
+        process_params.add(param)
+    if len(process_params) > 1:
+        raise Exception('Cost accounts do not use the same process parameter.')
```

Comment:
1. This should be a `ValueError`
2. There is probably a more efficient way to test this than to build a set. Could you not do something like:

```
param_check = None
for account in cost_accounts:
    param = BB_costing_exponents[str(tech)][account]['Process Parameter']

    if param_check is None:
        param_check = param
    elif param != param_check:
        raise ValueError()
```